src/Microsoft.Health.Fhir.Core/Features/Conformance/Models/DefaultOptionHashSet.cs

```diff
@@ -21,7 +21,7 @@ public DefaultOptionHashSet(T defaultOption, IEqualityComparer<T> comparer)
             DefaultOption = defaultOption;
         }
 
-        public T DefaultOption { get; }
+        public T DefaultOption { get; set; }
```

Comment:
I think add this to `IDefaultOption` also, it will make the usage code nicer in this case

Reply:
From what I can tell that won't make a difference in the usage as you still need to implement the DefaultOption property in the DefaultoptioHashSet. Plus in the interface, DefaultOption is an object, while in the DefaultOptionHashSet it is of type T. Unless I'm missing something, you would still need to cast to "DefaultOptionHashset<string>" since the Versioning in the ListedResourceComponent is of type ICollection<string>.

Reply:
Got it, sounds ok for now 👍